.github/workflows/tests.yaml

```diff
@@ -66,6 +66,9 @@ jobs:
       - name: 'clean up zunit & revolver'
         run: rm -rf revolver zunit
 
+      - name: "ices"
+        run: zunit run tests/ices.zunit
+
```

Comment:
Nit: the tests are sorted alphabetically

Reply:
fixed